dartagnan/src/main/java/com/dat3m/dartagnan/wmm/definition/SyncWith.java

```diff
@@ -0,0 +1,16 @@
+package com.dat3m.dartagnan.wmm.definition;
+
+import com.dat3m.dartagnan.wmm.Definition;
+import com.dat3m.dartagnan.wmm.Relation;
+
+public class SyncWith extends Definition {
```

Comment:
Please add some documentation about what this relation is

Reply:
Added